docs/website/src/main/resources/certifications/jakarta-annotations/1.3/TCK-Results.md

```diff
@@ -0,0 +1,36 @@
+TCK Results
+===========
+
+As required by the
+[Eclipse Foundation Technology Compatibility Kit License](https://www.eclipse.org/legal/tck.php),
+following is a summary of the TCK results for releases of Jakarta Annotations.
+
+# 1.3.4 Certification Request
+
+- Product Name, Version and download URL (if applicable) \
+  [Eclipse GlassFish 5.1](https://www.eclipse.org/downloads/download.php?file=/glassfish/glassfish-5.1.0.zip)
```

Comment:
I thought we were going to host these since they were all released before, we have a place to put them. Not all projects have web-sites. Ideally, the project sites would host them. May I resolve this?

Reply:
Hosting them here is ok even if not ideal.